numcodecs/tests/test_lzma.py

```diff
@@ -45,4 +54,7 @@ def test_config():
         check_config(codec)
 
     def test_repr():
-        check_repr('LZMA(format=1, check=0, preset=1, filters=None)')
+        if _py2:
+            check_repr('LZMA(format=1, check=0L, preset=1, filters=None)')
```

Comment:
Is this conditional logic needed for PY2? On my local system ``check_repr('LZMA(format=1, check=0, preset=1, filters=None)')`` works ok for both PY2 and PY3.